specification/marketplacecatalog/resource-manager/Solutions/preview/2023-03-01-preview/solutions.json

```diff
@@ -0,0 +1,501 @@
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Solution"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetUserSolution": {
+            "$ref": "./examples/GetUserSolution.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "UserSolutions"
+        ],
+        "operationId": "UserSolution_CreateOrUpdate",
+        "description": "Creates or updates a user solutions",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/CreateOrUpdateSolutionRequestPayload"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "CreateUserSolution": {
+            "$ref": "./examples/CreateUserSolution.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Solution"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "UserSolutions"
+        ],
+        "operationId": "RemoveUserSolution",
+        "description": "Removes a user solution",
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "RemoveUserSolution": {
+            "$ref": "./examples/RemoveUserSolution.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/mysolutions/{solutionId}/products/{productId}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "#/parameters/SolutionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ProductIdPathParameter"
+        }
+      ],
+      "put": {
+        "tags": [
+          "UserSolutions"
+        ],
+        "operationId": "UserSolutionProduct_Add",
+        "description": "Adds a product to a user solution",
+        "responses": {
+          "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=11&length=3)

this API is missing a request and response body

Reply:
removed this endpoint